app/src/main/res/layout/account_list_layout.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+              android:id="@+id/llAccountItem"
+              android:orientation="vertical"
+              android:layout_width="match_parent"
+              android:layout_height="wrap_content">
+
+    <TextView
+            android:id="@+id/tvAccountName"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:textSize="18sp"
+            android:textColor="#000000"
+            android:layout_marginRight="16dp"
```

Comment:
Replace 16dp, 8 dp, etc. by some variables call like `smallMargin`.

[It is example](https://gist.github.com/InsanusMokrassar/190dd267721a5281ea63a842558132a2) of the default values. You can copypaste it and use in project

Reply:
Done